packages/react-router/tests/redirects.test.tsx

```diff
@@ -516,6 +516,7 @@ describe('after redirect, the route context is passed to beforeLoad', () => {
     }
     const router = createContextRouter(['/'], 'BEFORE_LOAD', ctx)
     await router.load()
+    await router.invalidate()
```

Comment:
These tests can probably be removed since they no longer represent what's happening in the "real-world".